apps/src/templates/instructions/ScrollButtons.jsx

```diff
@@ -107,95 +118,103 @@ class ScrollButtons extends React.Component {
       ? this.props.height > 100
       : this.props.height > 60;
 
-    let upStyle = {
-      opacity: this.props.visible ? 1 : 0,
-      top: '10px',
-      margin: '0 0 3px 0',
-      left: centerItems ? '50%' : 25,
-      transform: 'translateX(-50%)',
-    };
-
-    const downStyle = {
-      opacity: this.props.visible ? 1 : 0,
-      bottom: '10px',
-      right: centerItems ? '50%' : 25,
-      transform: 'translateX(50%)',
-    };
-
-    const minecraftButton = {
-      width: 40,
-    };
-
-    const containerStyle = {
-      height: this.props.height,
-    };
-
     // for most tutorials, we have minimalist arrow elements. For
     // minecraft, we use a special button element to stylistically align
     // with the other buttons on the screen.
 
     const upButton = this.props.isMinecraft ? (
       <button
         type="button"
-        className="arrow"
+        className={classNames(
+          moduleStyles.up,
+          moduleStyles.minecraftButton,
+          this.props.visible && moduleStyles.visible,
+          centerItems && moduleStyles.upCenter,
+          'arrow'
+        )}
         ref={c => {
           this.scrollUp = c;
         }}
         key="scrollUp"
-        onMouseDown={this.scrollStartUp}
-        style={[styles.all, upStyle, minecraftButton]}
+        onClick={this.singleScrollUp}
+        onMouseDown={this.continuousScrollStartUp}
       >
         <img src="/blockly/media/1x1.gif" className="scroll-up-btn" alt="" />
       </button>
     ) : (
-      <div
+      <button
+        type="button"
         ref={c => {
           this.scrollUp = c;
         }}
         key="scrollUp"
-        onMouseDown={this.scrollStartUp}
-        style={[styles.all, styles.arrowGlyph, upStyle]}
+        onClick={this.singleScrollUp}
+        onMouseDown={this.continuousScrollStartUp}
+        className={classNames(
```

Comment:
Nit: could you order classNames similar to how you ordered for `isMinecraft` so it's easier to compare?

Reply:
Good call, done! Hopefully this is easier, LMK if not.

Reply:
Thanks!